html-lists/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HTML Lists</title>
+  </head>
+  <body>
+    <div>
+      <h1>HTML Lists</h1>
+      <h2>Ordered List Example</h2>
+        <ol>
+          <li>Learn to code.</li>
+          <li>Build projects.</li>
+          <li>Get a job.</li>
+        </ol>
```

Comment:
These don't need to be indented like this, the `ol` is a sibling of `h2`, so they should be at the same level of indentation.